README.md

```diff
@@ -71,6 +72,32 @@ This is all very experimental and incomplete, but we do have some basic examples
 functions and datatypes in Rust and manipulating them from Kotlin, Swift, and Python.
 Take a look in the [`./examples/`](./examples/) directory to see them in action.
```

Comment:
The `Status` section above here feels like it could maybe just be removed, in favour of the updated status that you added earlier in the doc.